install

```diff
@@ -162,8 +162,8 @@ group_chmods = %w( bin etc Frameworks include lib sbin share var
                    map { |d| File.join(HOMEBREW_PREFIX, d) }.select { |d| chmod?(d) }
 # zsh refuses to read from these directories if group writable
 zsh_dirs = %w( share/zsh share/zsh/site-functions )
-user_chmods = zsh_dirs.
-                  map { |d| File.join(HOMEBREW_PREFIX, d) }.select { |d| chmod?(d) }
+                  .map { |d| File.join(HOMEBREW_PREFIX, d) }
```

Comment:
Hi, I merged this to the latest state in the repo and fixed the coding style. Please check. Regarding your initial concern: if these directories don't exist, the script would create them. As for the situation when the permissions are already ok, I think it wouldn't hurt to apply them again.